Doc/library/shutil.rst

```diff
@@ -370,6 +395,28 @@ Directory and files operations
    operation. For :func:`copytree`, the exception argument is a list of 3-tuples
    (*srcname*, *dstname*, *exception*).
 
+.. _shutil-platform-dependent-efficient-copy-operations:
+
+Platform-dependent efficient copy operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting from Python 3.8 all functions involving a file copy (:func:`copyfile`,
+:func:`copy`, :func:`copy2`, :func:`copytree`, and :func:`move`) use
+platform-specific "fast-copy" syscalls in order to copy the file more
```

Comment:
Initially I phrased this as "zero-copy" because that's what `os.sendfile()` actually does under the hoods. Then it turns out Windows does something different (see https://github.com/python/cpython/pull/7160#discussion_r191640338). As such it made more sense to stick with the generic "fast-copy" term.  =)